File: common/src/test/java/net/cliff3/maven/common/util/NumberUtilTest.java
package net.cliff3.maven.common.util;

import static org.testng.Assert.*;

import lombok.extern.slf4j.Slf4j;
import org.testng.Assert;
import org.testng.annotations.Test;

/**
 * NumberUtilTest
 *
 * @author JoonHo Son
 * @since 1.0.0
 */
@Slf4j
public class NumberUtilTest {
    @Test
    public void getRandomIntegerTest() {
    	final int min = 11;
    	final int max = 30;
    	int result;

        for (int i = min; i <= max; i++) {
            result = NumberUtil.getRandomInteger(min, max);

            assertTrue(result >= min && result <= max, "임의의 정수 추출 실패");
        }
    }
}

File: common/src/test/java/net/cliff3/maven/common/util/NumberUtilTest.kt
package net.cliff3.maven.common.util

import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.testng.annotations.Test
import kotlin.test.assertTrue

/**
 * NumberUtilTest
 *
 * @author JoonHo Son
 * @version 1.0.0 2019-08-30 최초 작성
 * @since 1.0.0
 */
class NumberUtilTest {
    private val logger: Logger = LoggerFactory.getLogger(NumberUtilTest::class.java)

    @Test
    fun getRandomIntegerTest() {
        val min = 10
        val max = 100
        var result:Int

        for (x in 0..100) {
            result = NumberUtil.getRandomInteger(min, max)

            logger.debug("result -> {}", result)

            assertTrue(result >= 10, "최소값 보다 작음")
            assertTrue(result <= 100, "최대값 보다 큼")
        }
    }
}